tests/test_incidents.py

```diff
@@ -442,3 +438,75 @@ def test_gitea_incidents() -> None:
             assert s["INCIDENT_REPO"] == f"{expected_repo}-{arch}/"
             assert s["REPOHASH"] == repo_hash
             assert s["VERSION"] == product_ver
+
+
+def test_handle_incident_git_not_ongoing() -> None:
+    inc_data = {
+        "number": 123,
+        "rr_number": 1,
+        "project": "SUSE:Maintenance:123",
+        "inReview": True,
+        "isActive": False,
+        "inReviewQAM": False,
+        "approved": True,
+        "embargoed": False,
+        "packages": ["foo"],
+        "channels": ["SUSE:Updates:SLE-Product-SLES:15-SP3:x86_64"],
+        "emu": False,
+        "type": "git",
+    }
+    inc = Incident(inc_data)
+
+    test_config = {"FLAVOR": {"AAA": {"archs": [""], "issues": {}}}}
+    incidents_obj = Incidents(
+        product="",
+        product_repo=None,
+        product_version=None,
+        settings={"VERSION": "", "DISTRI": None},
+        config=test_config,
+        extrasettings=None,
+    )
+
+    ctx = IncContext(inc=inc, arch="", flavor="AAA", data={})
+    cfg = IncConfig(token={}, ci_url=None, ignore_onetime=False)
+
+    result = incidents_obj._handle_incident(ctx, cfg)  # noqa: SLF001
+    assert result is None
+
+
+def test_handle_incident_with_ci_url(mocker: MockerFixture) -> None:
+    inc_data = {
+        "number": 123,
+        "rr_number": 1,
+        "project": "SUSE:Maintenance:123",
+        "inReview": True,
+        "isActive": True,
+        "inReviewQAM": True,
+        "approved": False,
+        "embargoed": False,
+        "packages": ["foo"],
+        "channels": ["SUSE:Updates:SLE-Product-SLES:15-SP3:x86_64"],
+        "emu": False,
+        "type": "smelt",
+    }
+    inc = Incident(inc_data)
+
+    test_config = {"FLAVOR": {"AAA": {"archs": ["x86_64"], "issues": {"OS_TEST_ISSUES": "SLE-Product-SLES:15-SP3"}}}}
```

Comment:
suggestion: maybe more readable like this?

```suggestion
    test_config = {
        "FLAVOR": {
            "AAA": {
                "archs": ["x86_64"],
                "issues": {
                    "OS_TEST_ISSUES": "SLE-Product-SLES:15-SP3"
                }
            }
        }
    }
```

Add more lines, but I would prefer readability over length.